.travis.yml

```
@@ -4,19 +4,13 @@ matrix:
   include:
     - php: 7.2
       dist: bionic
-      env: COMPOSER_OPTS=""
     - php: 7.3
       dist: bionic
-      env: COMPOSER_OPTS=""
     - php: 7.4
       dist: bionic
-      env: COMPOSER_OPTS=""
     - php: nightly
       dist: bionic
       env: COMPOSER_OPTS="--ignore-platform-reqs"
```

i'd say we limit this the only requirement we really want to disable
```suggestion
      env: COMPOSER_OPTS="--ignore-platform-req=php"
```